spec/lib/farscape/plugins_spec.rb

```diff
@@ -223,6 +224,64 @@ def call(env)
 
     end
 
+    context 'extensions' do
+      before(:each) { Farscape.clear }
+      after(:all) { Farscape.clear }
+        
+      let(:entry_point) { "http://localhost:#{RAILS_PORT}"}
+      let(:can_do_hash) { {conditions: 'can_do_anything'} }
+      
+      it 'allows extensions' do
+        module Peacekeeper
+          def pacify!
+            raise 'none shall pass' if enabled_plugins.include?(:Peacekeeper)
```

Comment:
Interesting, did we talk about this? Why is the plugin responsible for checking whether it's enabled?